lib/response.js

```diff
@@ -204,7 +252,6 @@ class Response {
     if (!this.headerSent) {
       if (this._chunked) {
         this.setHeader('Content-Length', n)
-        this._chunked = false
```

Comment:
why are these gone?

Reply:
this is not necessary anymore because of the `this._chunked = false` now is the responsibility of the `setHeader` which check for the `Content-Length` key and set to false the _chunked